chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -36,6 +36,14 @@
             <name>Inherited Certification Status Errors Report</name>
             <group>chplJobs</group>
         </delete-job>
+        <delete-job>
+            <name>Inherited Certification Status Errors Report</name>
```

Comment:
It looks like this Inherited Certification Status Errors got put in here twice, but actually I think you can remove everything else that is being deleted (those changes already in prod) and only add the lines for your change.